### PR TITLE
test(e2e): Test installation for all templates

### PR DESCRIPTION
**Pull request by francoischalifour**
This PR adds more confidence by asserting that all templates get installed correctly (with Node and iOS templates).

### CODE REVIEW

File: scripts/e2e-installs.test.js [hunk 1]
```diff
@@ -27,31 +34,33 @@ describe('Installation', () => {
...
+        describe(templateName, () => {
+          test('get installed by default', () => {
+            execSync(
+              `yarn start ${appPath} \
+                --template "${templateName}"`,
+              { stdio: 'ignore' }
+            );
+
+            const dependenciesDirectory =
+              templateName === 'InstantSearch iOS' ? 'Pods' : 'node_modules';
```
Comment (samouss) [inline scripts/e2e-installs.test.js]: I would rather keep the tests a bit more simpler. If you want to tests all the templates installation, maybe scope them by categories (web, mobile)? But we could avoid to put too much logic in the test.

Comment (francoischalifour) [reply scripts/e2e-installs.test.js]: I refactored it. Let me know if it's a bit better now.

File: scripts/e2e-installs.test.js [hunk 2]
```diff
@@ -27,29 +28,51 @@ describe('Installation', () => {
...
+        'JavaScript Client',
+        'JavaScript Helper',
+        'React InstantSearch',
+        'React InstantSearch Native',
+        'Vue InstantSearch',
+      ]
+        .map(name => path.join(templatesFolder, name))
+        .filter(source => fs.lstatSync(source).isDirectory());
+
+      templates.map(templateName => templateName).forEach(templatePath => {
```
Comment (samouss) [inline scripts/e2e-installs.test.js]: The `map` is a bit useless right?

Comment (francoischalifour) [reply scripts/e2e-installs.test.js]: Oops, forgot to remove it!

**Pull request merged**